apis/v1alpha1/instrumentation_types.go

```diff
@@ -218,6 +222,35 @@ type ApacheHttpd struct {
 	// Needed only if different from default "/usr/local/apache2/conf"
 	// +optional
 	ConfigPath string `json:"configPath,omitempty"`
+
+	// Resources describes the compute resource requirements.
+	// +optional
+	Resources corev1.ResourceRequirements `json:"resourceRequirements,omitempty"`
+}
+
+// Nginx defines Nginx SDK and instrumentation configuration.
+type Nginx struct {
+	// Image is a container image with Nginx SDK and auto-instrumentation.
+	// +optional
+	Image string `json:"image,omitempty"`
```

Comment:
Is there already image for nginx instrumentation?

Reply:
ah it's here https://github.com/open-telemetry/opentelemetry-operator/pull/1852

Reply:
Here's the Nginx Otel module: https://github.com/open-telemetry/opentelemetry-cpp-contrib/blob/main/instrumentation/nginx

Reply:
For Nginx autoinstrumentation, default image with the instrumentation libraries is the same as for Apache Httpd - autoinstrumentation-apache-httpd